packages/pluggableWidgets/slider-web/src/Slider.editorConfig.ts

```diff
@@ -0,0 +1,136 @@
+import { hidePropertiesIn, hidePropertyIn, Problem, Properties } from "@mendix/piw-utils-internal";
+import { MaxValueTypeEnum, MinValueTypeEnum, SliderPreviewProps, StepSizeTypeEnum } from "../typings/SliderProps";
+
+const keysToHideByMinValueType: Record<MinValueTypeEnum, Array<keyof SliderPreviewProps>> = {
+    static: ["minAttribute", "expressionMinimumValue"],
+    dynamic: ["staticMinimumValue", "expressionMinimumValue"],
+    expression: ["staticMinimumValue", "minAttribute"]
+};
+
+const keysToHideByMaxValueType: Record<MaxValueTypeEnum, Array<keyof SliderPreviewProps>> = {
+    static: ["maxAttribute", "expressionMaximumValue"],
+    dynamic: ["staticMaximumValue", "expressionMaximumValue"],
+    expression: ["staticMaximumValue", "maxAttribute"]
+};
+
+const keysToHideBySizeType: Record<StepSizeTypeEnum, Array<keyof SliderPreviewProps>> = {
+    static: ["stepAttribute", "expressionStepSize"],
+    dynamic: ["stepValue", "expressionStepSize"],
+    expression: ["stepValue", "stepAttribute"]
+};
+
+export function getProperties(values: SliderPreviewProps, defaultProperties: Properties): Properties {
+    hidePropertiesIn(defaultProperties, values, keysToHideByMinValueType[values.minValueType]);
+
+    hidePropertiesIn(defaultProperties, values, keysToHideByMaxValueType[values.maxValueType]);
+
+    hidePropertiesIn(defaultProperties, values, keysToHideBySizeType[values.stepSizeType]);
+
+    if (!values.showTooltip) {
+        hidePropertyIn(defaultProperties, values, "tooltip");
+    }
+
+    return defaultProperties;
+}
+
+type CheckFn = (props: SliderPreviewProps) => Problem | undefined;
+
+const minValueCheck: CheckFn = ({ minValueType, expressionMinimumValue, minAttribute }) => {
+    if (minValueType === "expression" && expressionMinimumValue === "") {
+        return {
+            property: "expressionMinimumValue",
+            message: "Minimum value expression is empty"
+        };
+    }
+
+    if (minValueType === "dynamic" && minAttribute === "") {
+        return {
+            property: "minAttribute",
+            message: "Missing attribute for 'Minimum value' property"
+        };
+    }
+};
+
+const maxValueCheck: CheckFn = ({ maxValueType, expressionMaximumValue, maxAttribute }) => {
+    if (maxValueType === "expression" && expressionMaximumValue === "") {
+        return {
+            property: "expressionMaximumValue",
+            message: "Maximum value expression is empty"
+        };
+    }
+
+    if (maxValueType === "dynamic" && maxAttribute === "") {
+        return {
+            property: "maxAttribute",
+            message: "Missing attribute for 'Maximum value' property"
+        };
+    }
+};
+
+const minMaxValueCheck: CheckFn = ({ minValueType, maxValueType, staticMinimumValue, staticMaximumValue }) => {
+    const isPossibleToCheck = minValueType === "static" && maxValueType === "static";
+
+    if (!isPossibleToCheck) {
```

Comment:
Extracting to a const is good for readability, but since you're also negating it and it's only used once, I feel like the added value is limited. Instead, just doing the following should be fine:
```
if (minValueType !== "static" || maxValueType !== "static") {
    return;
}
```